<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <rosparam command="load" file="$(find robotx_recognition)/config/hsv_buoy_detector_config.yaml" ns="hsv_buoy_detector_node"/>

  <node name="hsv_buoy_detector_node" pkg="robotx_recognition" type="hsv_buoy_detector_node" args="" output="screen" respawn="true">
    <remap from="/hsv_buoy_detector_node/image_raw" to="/wam_v/front_camera/image_raw"/> 
  </node>
</launch>
